src/Aeon.Acquisition/GetRepository.cs

```diff
@@ -0,0 +1,37 @@
+﻿using Bonsai;
+using System;
+using System.ComponentModel;
+using System.Reactive.Linq;
+using LibGit2Sharp;
+
+
+namespace Aeon.Acquisition
+{
+    [DefaultProperty(nameof(RepoPath))]
+    [Description("Returns a Repository object from the target repository root path. " +
```

Comment:
Can we remove the `+` for now? Just for consistency with the rest of the code but we can discuss the best practice for this in the future later.

We could probably clarify the description adapting from LibGit2Sharp documentation as:

"Creates a repository object from the specified working directory or .git folder."